docker-compose.yml

```diff
@@ -4,24 +4,14 @@ services:
 
   backend:
     build:
-      context: .
-      dockerfile: ./backend/Dockerfile
+      context: ./backend
```

Comment:
The `context` here can be where the `Dockerfile` for the service actually is. Simplifies a bit some of the path-ing. 

For instance, the `dockerfile` value now doesn't need specifying as the `Dockerfile` inside the directory `./backend` is automatically used (though if there are multiple `Dockerfile`s in the same directory, then specifying is needed). 

Also, in the `Dockerfile` itself, the `COPY` command doesn't need the `./backend` path anymore as the image is now being built from the context `./backend`

---

Same applies for the frontend

Reply:
Thanks for the detailed explanation, @wkyoshida!